### PR TITLE
Feature/kas 4102 tweak signatures tab

### PR DESCRIPTION
**Pull request by sergiofenoll**
https://kanselarij.atlassian.net/browse/KAS-4102

Updates the layout of the signatures overview

### CODE REVIEW

File: app/templates/signatures/ongoing.hbs [hunk 1]
```diff
@@ -20,22 +20,30 @@
...
       @onChange={{fn (mut this.sort)}}
     />
     <th>{{t "status"}}</th>
+    <th></th>
   </:header>
   <:body as |signFlow|>
     {{#let signFlow.signSubcase.signMarkingActivity as |signMarkingActivity|}}
       {{#let signMarkingActivity.piece as |piece|}}
-        <td>
+        <td class="au-u-flex au-u-flex--spaced-tiny">
```
Comment (ValenberghsSven) [inline app/templates/signatures/ongoing.hbs]: the flex here messes with the rows for me.
image
Longer titles mean a large column size, but lots of whitespace
image

File: app/templates/signatures/ongoing.hbs [hunk 2]
```diff
@@ -59,6 +76,29 @@
...
+              {{t "open"}}
+            </AuLinkExternal>
+          {{else}}
+            <AuButton
+              @skin="naked"
+              @icon="chevron-right"
+              @hideText={{true}}
+              @disabled={{true}}
+            >
+              {{t "open"}}
```
Comment (ValenberghsSven) [inline app/templates/signatures/ongoing.hbs]: Is this a placeholder?
Perhaps a different button icon & tooltip to indicate the link to signinghub is not working (maybe you have no rights or it got removed from signinghub)

File: app/controllers/signatures/ongoing.js [hunk 3]
```diff
@@ -20,4 +22,41 @@ export default class SignaturesOngoingController extends Controller {
...
+    const signSigningActivities = await signSubcase.signSigningActivities;
+    const mandatees = await Promise.all(
+      signSigningActivities.map((activity) => activity.mandatee)
+    );
+    const persons = await Promise.all(
+      mandatees
+        .sort((m1, m2) => m1.priority - m2.priority)
+        .map((mandatee) => mandatee.person)
+    );
+    return persons.map((person) => person.fullName);
```
Comment (ValenberghsSven) [inline app/controllers/signatures/ongoing.js:44]: I think this is right? For agenda it makes sense to show them based on priority.
For signatures, is the `requestBy` more important or not? Leave as-is for now.

**Review by ValenberghsSven — CHANGES_REQUESTED** [review]
Small points:

1. ongoing.hbs still has `{{page-title (t "ongoing")}}`, needs to be "follow-up"?
2. The name in router.js is still "wordt-verwerkt". Needs changing?
3. the index route has no name. Does that need to be "handtekeningen/op-te-starten" or not? It made sense not to give it a name when there was only 1 subroute.
4. Button that does nothing (but looks the same like the normal one)

File: app/templates/signatures/ongoing.hbs (hunk 1 above)
Comment (ValenberghsSven) [reply app/templates/signatures/ongoing.hbs]: with pill
image

File: app/templates/signatures/ongoing.hbs (hunk 2 above)
Comment (ValenberghsSven) [reply app/templates/signatures/ongoing.hbs]: I changed the icon to a "broken-link".

File: app/templates/signatures/ongoing.hbs (hunk 1 above)
Comment (ValenberghsSven) [reply app/templates/signatures/ongoing.hbs]: removed the flex from the column, looks better

**Comment by ValenberghsSven** [comment]
> Small points:
> 
>     1. ongoing.hbs still has `{{page-title (t "ongoing")}}`, needs to be "follow-up"?
> 
>     2. The name in router.js is still "wordt-verwerkt". Needs changing?
> 
>     3. the index route has no name. Does that need to be "handtekeningen/op-te-starten" or not? It made sense not to give it a name when there was only 1 subroute.
> 
>     4. Button that does nothing (but looks the same like the normal one)

1. changed it (checked prototype)
2. changed it
3. added dutch name for index to `router.js`
4. changed the icon so it's different from the actual link

**Review by ValenberghsSven — APPROVED** [review]
Despite my own minor changes the main work is good

**Review by tdn — APPROVED** [review]
Approving this, since we'll have to elaborate on this view in the future anyway

**Pull request merged**